style.css

```diff
@@ -62,3 +62,489 @@ nav {
   line-height: 24px;
   background: #0e0e0f;
 }
+
+.text3 {
+  position: absolute;
+  width: 181px;
+  left: 16px;
+  top: 650px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.projectpageimage1 {
+  background: url("Rectangle_21.svg");
```

Comment:
- [ ] Please check your background image is repeating in the document to try to fix this issue. 
Hint:- use of `no-repeat` property.

![image](https://user-images.githubusercontent.com/53019916/125153345-ad98ef80-e170-11eb-9f8b-cc3af793a4d5.png)